### PR TITLE
Handle none existing file

### PR DESCRIPTION
**Pull request by lloricode**

### CODE REVIEW

File: src/Http/Controllers/SwaggerController.php [hunk 1]
```diff
@@ -72,6 +72,10 @@ public function docs(Request $request, string $file = null)
             }
         }
 
+        if (! File::exists($filePath)) {
```
Comment (DarkaOnLine) [inline src/Http/Controllers/SwaggerController.php:75]: But the file should be created on line 60 and if not we have an error for that. What was a situation you encountered that file was not created?

Comment (lloricode) [reply src/Http/Controllers/SwaggerController.php:75]: Hi, just try to access the json file directly to browser, then try to input some not existing file will return 500 status

Comment (DarkaOnLine) [reply src/Http/Controllers/SwaggerController.php:75]: Ah, that makes sense. Can you add the error message something like `Documentation not found`? And also add the test for this?

Comment (lloricode) [reply src/Http/Controllers/SwaggerController.php:75]: Yes I can, I'll be right back :)

**Review by Messhias — CHANGES_REQUESTED** [review]
@lloricode the style ci isn't not passing by, you need to do some changes.

**Comment by DarkaOnLine** [comment]
Fixed in #385

**Pull request closed without merging**